examples/latex.jl

```diff
@@ -15,13 +15,14 @@ function main(window)
              textinput() >>> texᵗ,
              hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
```

Comment:
Checkbox doesn't actually do anything  Version 0.4.6-pre+18 (other than toggle the checkbox on/off)